cdc/entry/schema_storage.go

```diff
@@ -689,7 +690,9 @@ func (s *SchemaStorage) getSnapshot(ts uint64) (*schemaSnapshot, error) {
 // GetSnapshot returns the snapshot which of ts is specified
 func (s *SchemaStorage) GetSnapshot(ctx context.Context, ts uint64) (*schemaSnapshot, error) {
 	var snap *schemaSnapshot
-	err := retry.Run(10*time.Millisecond, 25,
+	// The infinite retry here is a temporary solution to the `ErrSchemaStorageUnresolved` caused by
+	// DDL puller lagging too much.
+	err := retry.Run(10*time.Millisecond, math.MaxInt64,
```

Comment:
The default `ExponentialBackOff` config also has a `MaxElapsedTime` restriction, which has a default value `15min`.

Besides, do we need to add some log if the retry round is large enough, such as after the 20 retries, log a warning for each retry? If not, users could have to investigate the Grafana metrics to find why the changefeed checkpoint does not update.